src/ProtobufProtocolSupport.chpl

```diff
@@ -645,6 +645,73 @@ module ProtobufProtocolSupport {
       return s;
     }
 
+    record Any {
+      var typeUrl: string;
+      var value: bytes;
+
+      proc packFrom(messageObj) throws {
+        var s: bytes;
+        var tmpMem = openmem();
+        var memWriter = tmpMem.writer(kind=iokind.little, locking=false);
+        var memReader = tmpMem.reader(kind=iokind.little, locking=false);
+
+        messageAppend(messageObj, 2, memWriter);
+        memWriter.close();
+        memReader.readbytes(s);
+        tmpMem.close();
+
+        this.value = s;
+        this.typeUrl = getTypeUrl(messageObj);
+      }
+
+      proc unpackTo(ref messageObj) throws {
+        var url = getTypeUrl(messageObj);
+        if (url != this.typeUrl) {
+          throw new owned IllegalArgumentError("input message type is not same as global identifier");
```

Comment:
reword to maybe "input message type does not match destination message type"

Reply:
Yeah sure!